src/features/session_replay/shared/stylesheet-evaluator.js

```diff
@@ -19,15 +19,15 @@ class StylesheetEvaluator {
     let incompletes = 0
     if (isBrowserScope) {
       for (let i = 0; i < Object.keys(document.styleSheets).length; i++) {
```

Comment:
This could be more concisely wrote as a forEach loop since we are not breaking out of the loop early.